cli/src/main/java/com/devonfw/tools/ide/tool/cobigen/Cobigen.java

```diff
@@ -0,0 +1,32 @@
+package com.devonfw.tools.ide.tool.cobigen;
+
+import java.util.Set;
+
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.tool.LocalToolCommandlet;
+import com.devonfw.tools.ide.tool.ToolCommandlet;
+import com.devonfw.tools.ide.tool.java.Java;
+
+/**
+ * {@link ToolCommandlet} for cobigen CLI (cobigen).
+ */
+public class Cobigen extends LocalToolCommandlet {
+
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}.
+   */
+  public Cobigen(IdeContext context) {
+
+    super(context, "cobigen", Set.of(TAG_CLOUD, TAG_IAC));
```

Comment:
Cobigen is a codegenerator mainly for Java but also for TypeScript or potentially C#/.NET or other languages.
Meanwhile the PR #166 was merged so the Tags are not Strings anymore so this code needs to be updated.

Reply:
Changed to Tag.GENERATOR